### ISSUE TITLE
[3.0.8] Accessibility recommendations

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#23431]
**Describe the bug**
I am filing community report on accessibility from a reliable source. While I don't have the exact steps to reproduce, tracking and addressing this is the way forward.

1. "Participant list is still not adapted to 3.0 -> is not an html compliant list."
2. The focus trap from the welcome popup is still missing 
3. Text for emojis in response to a chat message is missing

Reported during BigBlueButton Community call on June 10, 2025

**BBB version:**
BigBlueButton 3.0.8

**Comment by JoVictorNunes** [comment]
> 3. Text for emojis in response to a chat message is missing

I couldn't reproduce nor tracking down what might be going on. Any clues?

**Comment by KDSBrowne** [comment]
I wasn't able to reproduce the issue either — the screen reader seems to have access to the emoji reactions in the chat messages. It might not be the most semantically accurate, though.

```
Jane  
19:36  
Hello  
button    👍  
button    1  
button    bob reacted with :+1:  
button    😱  
button    1  
button    john reacted with :scream:  
out of list  clickable  
```
